newIDE/app/src/ObjectGroupEditor/index.js

```diff
@@ -98,6 +96,7 @@ const ObjectGroupEditor = ({
             globalObjectsContainer={globalObjectsContainer}
             objectsContainer={objectsContainer}
             value={newObjectName}
+            excludedObjects={objectsInGroup}
```

Comment:
I think in theory this could be called, to be super explicit:

```suggestion
            excludedObjectOrGroupNames={objectsInGroup}
```